### PR TITLE
2.19.1 disable network traffic WIP

### PR DESCRIPTION
**Pull request by mjkkirschner**
This pulls in some changes from @pinzart90's changes for 2.17.4 and tries to adapt them to use the `DisableAnalytics` flag as a stand in `DisableNetworkActivity` flag until we have that.

Additionally, it disables the notification center requests if `DisableAnalytics` is true.

I believe it is safe to merge to master, unlike the original changes.

### CODE REVIEW

File: src/DynamoApplications/StartupUtils.cs [hunk 1]
```diff
@@ -411,6 +411,10 @@ private static bool PreloadASM(string asmPath, out string geometryFactoryPath, o
                 IsServiceMode = isServiceMode,
                 Preferences = PreferenceSettings.Instance
             };
+            config.AuthProvider = CLImode ? null : new Core.IDSDKManager();
```
Comment (sm6srw) [inline src/DynamoApplications/StartupUtils.cs:414]: It looks like the alias version of Sandbox should never get here. They have their own MakeModel implementation and are not calling any of ours as far as I can tell.

**Review by QilongTang — APPROVED** [review]

**Comment by QilongTang** [comment]
@mjkkirschner All changes looks good, this is no longer WIP right?

**Comment by mjkkirschner** [comment]
@QilongTang let me make a few last updates to this!

**Comment by mjkkirschner** [comment]
@QilongTang @sm6srw PTAL - I added some comments and also disabled the notifications center if the DisableAnalytics static props was set true.

**Review by sm6srw — APPROVED** [review]
LGTM!

**Comment by QilongTang** [comment]
> @QilongTang @sm6srw PTAL - I added some TODO comments and also disabled the notifications center if the DisableAnalytics static props was set true.
> 
> I think this should be safe to merge to master as well so we can stop doing these one off builds in the future.

Thanks, nicely done

**Comment by QilongTang** [comment]
@mjkkirschner The previous job does have tests passed and given we have no tests for notification center, I merged

**Pull request merged**